libpod/rootless_cni_linux.go

```diff
@@ -0,0 +1,320 @@
+// +build linux
+
+package libpod
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"path/filepath"
+	"runtime"
+
+	cnitypes "github.com/containernetworking/cni/pkg/types/current"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containers/podman/v2/libpod/define"
+	"github.com/containers/podman/v2/libpod/image"
+	"github.com/containers/podman/v2/pkg/util"
+	"github.com/containers/storage/pkg/lockfile"
+	"github.com/hashicorp/go-multierror"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var rootlessCNIInfraImage = map[string]string{
+	// Built from ../contrib/rootless-cni-infra
+	// TODO: move to Podman's official quay
+	"amd64": "ghcr.io/akihirosuda/podman-rootless-cni-infra:gd34868a13-amd64",
```

Comment:
@TomSweeneyRedHat Do you own the Quay? I want to make sure we have an action item to get this moved once it merges